server/routes/classes.ts

```diff
@@ -12,12 +13,16 @@ const storage = new Storage({
 const bucketName = 'classes-bucket'
 const bucket = storage.bucket(bucketName)
 
+const ERROR_CLASS_NOT_FOUND = 'Class not found'
+const ERROR_SERVER = 'Internal Server Error'
+
 const allowedMimeTypes = ['video/mp4', 'video/mpeg', 'video/quicktime']
 
 const upload = multer({
     storage: multer.memoryStorage(),
     limits: {
-        fileSize: 1024 * 1024 * 1024, // 1 GB
+        fileSize: 1024 * 1024 * 5, // 1 GB
+        //fileSize: 1024 * 1024 * 1024, // 1 GB
```

Comment:
Uncomment this line and delete the previous one to set a limit of 1GB and not 5MB.